modular/gater/auth_handler.go

```diff
@@ -342,3 +598,29 @@ func (g *GateModular) verifySignedContent(signedContent string, expectedDomain s
 
 	return nil
 }
+
+func (g *GateModular) verifySignedContentV2(signedContent string, expectedDomain string, expectedPublicKey string, expectedExpiryDate string) error {
+	pattern := `(.+) wants you to sign in with your BNB Greenfield account:\n*(.+)\n*Register your identity public key (.+)\n*URI: (.+)\n*Version: (.+)\n*Chain ID: (.+)\n*Issued At: (.+)\n*Expiration Time: (.+)`
```

Comment:
Consider make this pattern a constant if we do not change it often?

Reply:
Done